core/store/src/trie/trie_tests.rs

```diff
@@ -9,16 +9,14 @@ use near_primitives::hash::{hash, CryptoHash};
 use near_primitives::shard_layout::ShardUId;
 use rand::seq::SliceRandom;
 use rand::Rng;
-use std::cell::RefCell;
 use std::collections::{HashMap, HashSet};
 use std::fmt::Debug;
-use std::rc::Rc;
-use std::sync::Arc;
+use std::sync::{Arc, RwLock};
 
 /// TrieMemoryPartialStorage, but contains only the first n requested nodes.
 pub struct IncompletePartialStorage {
     pub(crate) recorded_storage: HashMap<CryptoHash, Arc<[u8]>>,
-    pub(crate) visited_nodes: RefCell<HashSet<CryptoHash>>,
+    pub(crate) visited_nodes: RwLock<HashSet<CryptoHash>>,
```

Comment:
Same here

Reply:
This on the other hand is just test code, so might very well be whatever.

Reply:
Oh yes I hadn’t checked